sycl/test/aot/Inputs/aot.cpp

```diff
@@ -0,0 +1,68 @@
+#include <CL/sycl.hpp>
```

Comment:
```suggestion
//==----- aot.cpp - Simple vector addition -----------------------------------==//
//
// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
// See https://llvm.org/LICENSE.txt for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
//
//===------------------------------------------------------------------------===//
#include <CL/sycl.hpp>
```

Reply:
Sure, I'll also perform more clean-up in the original test files